corehq/apps/userreports/templates/userreports/data_source_from_app.html

```diff
@@ -5,12 +5,12 @@
 
 {% block js-inline %}{{ block.super }}
     <script type="text/javascript">
-        $(function(){
-            ko.applyBindings({
-                optionsMap: ko.observable({{ data_source_options_map|JSON }}),
-                appId: ko.observable()
-            }, $('#data-source-config').get(0));
-        });
+        var viewModel = {
```

Comment:
small, but any reason for taking this outside of the anonymous function? `viewModel` is now a global for the page